<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1260</width>
    <height>962</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>四轴飞行器调试助手</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>0</y>
      <width>531</width>
      <height>151</height>
     </rect>
    </property>
    <property name="title">
     <string>历史消息</string>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <item>
      <widget class="QTextEdit" name="textEdit_history"/>
     </item>
    </layout>
   </widget>
   <widget class="QGroupBox" name="groupBox_9">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>681</width>
      <height>541</height>
     </rect>
    </property>
    <property name="title">
     <string>四轴模型-三维显示</string>
    </property>
    <widget class="QOpenGLWidget" name="openGLWidget">
     <property name="geometry">
      <rect>
       <x>-1</x>
       <y>19</y>
       <width>681</width>
       <height>521</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>OpenHandCursor</cursorShape>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_8">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>550</y>
      <width>1241</width>
      <height>371</height>
     </rect>
    </property>
    <property name="title">
     <string>波形图</string>
    </property>
    <widget class="TitleCurveWidget" name="widget1" native="true">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>401</width>
       <height>161</height>
      </rect>
     </property>
    </widget>
    <widget class="TitleCurveWidget" name="widget2" native="true">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>20</y>
       <width>401</width>
       <height>161</height>
      </rect>
     </property>
    </widget>
    <widget class="TitleCurveWidget" name="widget3" native="true">
     <property name="geometry">
      <rect>
       <x>830</x>
       <y>20</y>
       <width>401</width>
       <height>161</height>
      </rect>
     </property>
    </widget>
    <widget class="TitleCurveWidget" name="widget4" native="true">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>190</y>
       <width>401</width>
       <height>161</height>
      </rect>
     </property>
    </widget>
    <widget class="TitleCurveWidget" name="widget5" native="true">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>190</y>
       <width>401</width>
       <height>161</height>
      </rect>
     </property>
    </widget>
    <widget class="TitleCurveWidget" name="widget6" native="true">
     <property name="geometry">
      <rect>
       <x>830</x>
       <y>190</y>
       <width>401</width>
       <height>161</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>160</y>
      <width>200</width>
      <height>381</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>200</width>
      <height>0</height>
     </size>
    </property>
    <property name="title">
     <string>配置</string>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>端口号</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="comboBox_portname">
         <property name="maximumSize">
          <size>
           <width>92</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="editable">
          <bool>false</bool>
         </property>
         <property name="currentIndex">
          <number>-1</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>波特率</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="comboBox_baudrate">
         <property name="maximumSize">
          <size>
           <width>92</width>
           <height>16777215</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>数据位</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="comboBox_databits">
         <property name="maximumSize">
          <size>
           <width>92</width>
           <height>16777215</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>校验位</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="comboBox_parity">
         <property name="maximumSize">
          <size>
           <width>92</width>
           <height>16777215</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>停止位</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="comboBox_stopbits">
         <property name="maximumSize">
          <size>
           <width>92</width>
           <height>16777215</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_6">
       <item>
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>流控制</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="comboBox_flowcontrol">
         <property name="maximumSize">
          <size>
           <width>92</width>
           <height>16777215</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_openport">
       <property name="text">
        <string>打开</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_closeport">
       <property name="text">
        <string>关闭</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>930</x>
      <y>160</y>
      <width>308</width>
      <height>381</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>350</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="title">
     <string>参数读取</string>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QGroupBox" name="groupBox_4">
       <property name="title">
        <string>A组</string>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_7">
        <item>
         <widget class="QLabel" name="label_7">
          <property name="text">
           <string>加速度</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLCDNumber" name="lcdNumber_acc_A"/>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="label_8">
          <property name="text">
           <string>角加速度</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLCDNumber" name="lcdNumber_sig_A"/>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_5">
       <property name="title">
        <string>B组</string>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_8">
        <item>
         <widget class="QLabel" name="label_9">
          <property name="text">
           <string>加速度</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLCDNumber" name="lcdNumber_acc_B"/>
        </item>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="label_10">
          <property name="text">
           <string>角加速度</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLCDNumber" name="lcdNumber_sig_B"/>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_6">
       <property name="title">
        <string>C组</string>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_9">
        <item>
         <widget class="QLabel" name="label_11">
          <property name="text">
           <string>加速度</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLCDNumber" name="lcdNumber_acc_C"/>
        </item>
        <item>
         <spacer name="horizontalSpacer_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="label_12">
          <property name="text">
           <string>角加速度</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLCDNumber" name="lcdNumber_sig_C"/>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_7">
       <property name="title">
        <string>发送</string>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_10">
        <item>
         <widget class="QLineEdit" name="lineEdit_send"/>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_senddata">
          <property name="text">
           <string>发送</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </widget>
   <zorder>groupBox_3</zorder>
   <zorder>groupBox_2</zorder>
   <zorder>groupBox</zorder>
   <zorder>groupBox_9</zorder>
   <zorder>groupBox_8</zorder>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>TitleCurveWidget</class>
   <extends>QWidget</extends>
   <header location="global">titlecurvewidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
